android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -245,6 +251,34 @@ public void setBufferConfig(final ReactExoplayerView videoView, @Nullable Readab
         }
     }
 
+    @ReactProp(name = PROP_DRM_LICENSE_URL)
+    public void setDrmUrl(final ReactExoplayerView videoView, @Nullable String licenseUrl){
+        Log.d("setDrmUrl", licenseUrl);
+        videoView.setDrmLicenseUrl(licenseUrl);
+    }
+
+    @ReactProp(name = PROP_DRM_LICENSE_HEADER)
+    public void setDrmHeader(final ReactExoplayerView videoView, @Nullable ReadableMap headers){
+        ArrayList<String> drmKeyRequestPropertiesList = new ArrayList<>();
+        ReadableMapKeySetIterator itr = headers.keySetIterator();
+        while (itr.hasNextKey()) {
```

Comment:
There's no reason to convert this to an array, we can use a HashMap here to represent everything as key/value pairs. Let me know if you need an example.